### PR TITLE
Add projects/extensions

### PR DESCRIPTION
**Pull request by dillonbarnes**
Added are two very useful spotify-node-microbit projects and a handy MakeCode extension with different proejcets on.
The extension is still in development and I'm adding new projects quite often.
I think the community would really like these additions though

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

- [x] I have read and understood the [Contribution Guidelines](https://github.com/carlosperate/awesome-microbit/blob/master/contributing.md)
- [x] I am making an individual pull request for each suggestion
- [x] I have used the correct spelling and capitalisation for `BBC micro:bit`, `micro:bit`, and `Micro:bit Educational Foundation` (resource title could be excluded if there is a good reason)
- [x] The content linked is in English, or it contains an English translation
- [x] I have added the new entry to the bottom of its the category
- [x] I have used the following format:
```
- [Resource Title](link) - Resource short Description (2 lines or less in total)
```

### CODE REVIEW

**Comment by dillonbarnes** [comment]
@carlosperate 
Would the each Spotify program count as an individual one?

Thanks

**Comment by carlosperate** [comment]
If they are practically the same I'd combine them into a single entry. The "Official Android App" is an example of an entry doing something like that.

**Comment by dillonbarnes** [comment]
Ok.
I created a seperate pull request for the extension and deleted it from this one.
This one only has the two spotify projects now.

File: README.md [hunk 1]
```diff
@@ -436,7 +436,8 @@ All these projects contain steps and resources required for reproduction.
 - [Electromaker micro:bit projects](https://www.electromaker.io/projects?platform=microbit) - All the micro:bit projects posted to Electromaker, a platform for makers to showcase their projects.
 - [Saturday Science & BBC micro:bits](https://saturdayscience.org/bbc-microbit/) - Practical science and engineering projects with the micro:bit, explore physical properties with cool experiments.
 - [Maker Pro micro:bit Projects & Tutorials](https://maker.pro/microbit) - The micro:bit section of Maker Pro, a place for makers to share designs, collaborate, and learn how to take your product to market.
-
+- [microbit-node-and-spotify(Mac)](https://www.hackster.io/samelhusseini/micro-bit-spotify-mac-5ac6d7) - Allows you to control a Spotify playlist with 2 micro:bits on Mac.
+- [microbit-node-and-spotify(Windows)](https://www.hackster.io/samelhusseini/microbit-spotify-windows-8a8551) - Allows you to control a Spotify playlist with 2 micro:bits on Windows.
```
Comment (carlosperate) [inline README.md:440]: As mentioned in https://github.com/carlosperate/awesome-microbit/pull/368#issuecomment-673533506, could you combine these two into one?

Comment (dillonbarnes) [reply README.md:440]: I'll have a try. I id it like that as they have different links.

File: README.md [hunk 2]
```diff
@@ -437,7 +438,6 @@ All these projects contain steps and resources required for reproduction.
 - [Saturday Science & BBC micro:bits](https://saturdayscience.org/bbc-microbit/) - Practical science and engineering projects with the micro:bit, explore physical properties with cool experiments.
 - [Maker Pro micro:bit Projects & Tutorials](https://maker.pro/microbit) - The micro:bit section of Maker Pro, a place for makers to share designs, collaborate, and learn how to take your product to market.
 
-
```
Comment (carlosperate) [inline README.md:440]: Could you restore this line back please?

Comment (dillonbarnes) [reply README.md:440]: Sorry, but what do you mean?

File: README.md [hunk 3]
```diff
@@ -424,6 +424,7 @@ All these projects contain steps and resources required for reproduction.
 - [DIY Educational micro:bit Robot](https://www.instructables.com/id/DIY-Educational-Microbit-Robot/) - Building a relatively accessible, capable and cheap robot. Two variants provided with different sensors and example code for MakeCode and MicroPython.
 - [MIDI CC Wireless Controller](https://www.instructables.com/id/Microbit-Midi-CC-Wireless-Controller/) - A wireless MIDI CC controller, allowing you to use your micro:bit as a MIDI controller and connect it to your favourite music production software.
 - [Smart Garden Ornaments](https://github.com/jimbobbennett/smart-garden-ornaments) - Track things happening in your garden, neighbourhood, or school using smart ornaments with the BBC micro:bit, Raspberry Pi, and a cloud-based IoT service.
+- [microbit-node-and-spotify(Mac)](https://www.hackster.io/samelhusseini/micro-bit-spotify-mac-5ac6d7)[Windows](https://www.hackster.io/samelhusseini/microbit-spotify-windows-8a8551) - Allows you to control a Spotify playlist with 2 micro:bits.
```
Comment (carlosperate) [inline README.md:427]:
```suggestion
- [micro:bit + Spotify (macOS)](https://www.hackster.io/samelhusseini/micro-bit-spotify-mac-5ac6d7) - ([Windows version](https://www.hackster.io/samelhusseini/microbit-spotify-windows-8a8551)) Allows you to control a Spotify playlist with 2 micro:bits.
```

File: README.md [hunk 4]
```diff
@@ -437,7 +438,6 @@ All these projects contain steps and resources required for reproduction.
 - [Saturday Science & BBC micro:bits](https://saturdayscience.org/bbc-microbit/) - Practical science and engineering projects with the micro:bit, explore physical properties with cool experiments.
 - [Maker Pro micro:bit Projects & Tutorials](https://maker.pro/microbit) - The micro:bit section of Maker Pro, a place for makers to share designs, collaborate, and learn how to take your product to market.
```
Comment (carlosperate) [inline README.md:440]:
```suggestion


```

There was two new lines before the Articles section, now there is only one blank line in this PR. So we need to add an extra line.

Comment (dillonbarnes) [reply README.md]: Done.

**Pull request merged**